ByBit.Net/Clients/V5/BybitRestClientApiTrading.cs

```diff
@@ -239,7 +239,7 @@ internal BybitRestClientApiTrading(BybitRestClientApi baseClient)
             string? cursor = null,
             CancellationToken ct = default)
         {
-            if (orderId != null && clientOrderId != null)
```

Comment:
The check is correct, it's the error message which isn't right I think. Seeing as there are multiple situations with different restrictions maybe this check should just be removed

Reply:
I am fine with that, as Bybit will return an error if you send it without either, which is fine with me